src/silo/query_engine/exec_node/zstd_decompress_expression.cpp

```diff
@@ -29,15 +29,39 @@ struct BinaryDecompressKernel {
       const arrow::compute::ExecSpan& input,
       arrow::compute::ExecResult* out
    ) {
+      SPDLOG_INFO("BinaryDecompressKernel::Exec called!");
       if (input.num_values() != 2) {
          return arrow::Status::Invalid("Expected 2 input arrays, got ", input.num_values());
       }
 
       if (!input.values[0].is_array()) {
          return arrow::Status::Invalid("Expected array input, got scalar/chunked array");
       }
-      const auto& input_span = input.values[0].array;
-
+      const arrow::ArraySpan& input_span = input.values[0].array;
+      std::string buffer_debug_info{fmt::format(
+         "buffers (data/size): [{}/{}, {}/{}, {}/{}]",
+         fmt::ptr(input_span.buffers[0].data),
+         input_span.buffers[0].size,
+         fmt::ptr(input_span.buffers[1].data),
+         input_span.buffers[1].size,
+         fmt::ptr(input_span.buffers[2].data),
+         input_span.buffers[2].size
+      )};
```

Comment:
Do we still need this? If yes, it might be worth putting in a guard so that we only execute it when trace logs are active. (Otherwise we construct a lot of useless strings that will never get logged anyway).